packages/grid/helm/syft/templates/backend/backend-secret.yaml

```diff
@@ -11,6 +11,7 @@ data:
   defaultRootPassword: {{ include "common.secrets.set" (dict
     "secret" $secretName
     "key" "defaultRootPassword"
-    "default" .Values.node.defaultSecret.defaultRootPassword
+    "randomDefault" .Values.global.randomizedSecrets
```

Comment:
MAYBE: it would be good to setup application level secret randomization
instead of global.randomizedSecrets.

it could be 
.Values.global.node.randomizedSecrets


Where for example if users , need to set a custom secret only  for syft container, they  could do it easily.

Reply:
This could be done similar for  mongo, seaweedfs

Reply:
Hmm ... looks like it didn't get pushed